Common.xml

```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
       </Item>
       
       <Item ID="6044">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Battery Percentage</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
+        <Units>%</Units>
+        <Submitter>IoT Connectivity Alliance</Submitter>
+        <Description>Current remaining battery level</Description>
         <TS></TS>
         <TSLink>0</TSLink>
       </Item>
       
       <Item ID="6045">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
+        <Name>RSSI</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
         <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Units>dBW</Units>
+        <Submitter>IoT Connectivity Alliance</Submitter>
+        <Description>Received Signal Strength Indicator</Description>
         <TS></TS>
         <TSLink>0</TSLink>
       </Item>
       
       <Item ID="6046">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
+        <Name>Smoke Sensor State</Name>     
+        <Operations>R</Operations>     
+        <Type>Boolean</Type>
+        <RangeEnumeration>Normal (0) or Smoke detected (1)</RangeEnumeration>
```

Comment:
RangeEnumeration field should be machine-readable. See [Appendix D of LwM2M TS 1.1.1](http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.pdf). The explanation of values should be in description.

That said, for boolean values there’s only “true” and “false” value possible so RangeEnumeration does not provide much value here and can be left empty.

Reply:
What about change the Type to Enumeration?

Reply:
@QingAn - Your suggestion to change the Type to `Enumeration` can't be applied as this isn't a `Data Type` - [View Values](http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.pdf)

Did you want to update the `RangeEnumeration` to blank as suggested by Ari?

Reply:
@seanmcilroy29 Ok. Blank can work for me.